lib/dish.rb

```diff
@@ -0,0 +1,28 @@
+class Dish
+  attr_reader :dishes
+
+  def initialize
+    @dishes = []
+
+    preload_dishes
+  end
+
+  def print
+    return 'Puts no students at this time' if @dishes.empty?
+    i = 1
+    @dishes.each do |dish|
+      "#{i}. #{dish[:name].capitalize.ljust(24, '.')} #{dish[:price]}"
+      i += 1
+    end
+end
+
+  private
+
+  def preload_dishes
```

Comment:
You've hardcoded a set of dishes into this class. Is there another way of loading the menu's dishes?